stdlib/public/core/Int128.swift

```diff
@@ -307,8 +311,12 @@ extension Int128 {
   public func remainderReportingOverflow(
     dividingBy other: Self
   ) -> (partialValue: Self, overflow: Bool) {
-    _precondition(other != .zero, "Division by zero in remainder operation")
-    if self == .min && other == -1 { return (0, true) }
+    if _slowPath(other == .zero) {
+      return (self, true)
+    }
+    if _slowPath(self == .min && other == (-1 as Self)) {
```

Comment:
This case is somewhat interesting because the remainder _does not_ overflow; rather the analogous division does. It's entirely plausible to me that this was originally defined "wrong", but we're probably stuck with the current behavior. In any case, I think it merits a comment here.

Reply:
I return the equivalent of (0, false) in my project. So, I considered asking about this specific interaction, but I figured Swift chose C++ semantics ± trapping on undefined behavior. Is there a particular comment you have in mind?

Reply:
This was the specified behavior—I was the one who aligned the actual behavior with the specification in #14219—but for this reason the wording of the corresponding precondition failure (at least at the time) was that "_Division_ results in overflow in remainder operation"